### ISSUE TITLE
"contribution" used only in one section

### ISSUE THREAD

**Issue by tieguy** [ml5js/Code-of-Conduct#20]
"contribution" is used only in the reversion clause, and is not defined. Feels like it would be cleaner to replace it.

No super-clean way to solve given the lack of some key terms in Blue Oak Model License, but maybe something like "once three years have elapsed from the date the software is made available to the public" (language from #19)?

**Comment by mwweinberg** [comment]
I agree that it would be cleaner to replace it and like the language in #19

**Comment by mwweinberg** [comment]
Coming back to this (and in light of the resolution of #19), I'm still wondering if this is something that is fixable in an elegant way.  I worry that the "from from the date the software is made available to the public" language could effectively toll the 3 year period indefinitely because it would include ml5 as a whole.  I think that the way that this works is that the license recognizes that each individual contribution is a distinctly licensable element.
